src/components/DropZone/components/FileUpload/tests/FileUpload.test.tsx

```diff
@@ -102,18 +102,4 @@ describe('<FileUpload />', () => {
     fileUpload.setProps({children: <FileUpload />});
     expect(fileUpload.find(TextStyle).text()).toBe('or drop files to upload');
   });
-
-  it('does not use default action title and hint when props are changed', () => {
```

Comment:
This test failed in the conversion and I deleted it since we don't need it anymore. When this component was a class we stored this on state and this was a regression test however we don't follow that pattern in our functional component so 🔥